ethcore/src/engines/tendermint/mod.rs

```diff
@@ -37,11 +37,12 @@ use ethkey::{recover, public_to_address, Signature};
 use account_provider::AccountProvider;
 use block::*;
 use spec::CommonParams;
-use engines::{Engine, Seal, EngineError};
+use engines::{Engine, Seal, EngineError, ConstructedVerifier};
+use engines::authority_round::{combine_proofs, destructure_proofs};
```

Comment:
Also, for tendermint, `combine_proofs` signature can be `fn(&[u8], Header) -> Vec<u8>` and `destructure_proofs` would be `fn(&[u8]) -> Result<(&[u8], Header), _>`